lib/std/thread.zig

```diff
@@ -500,4 +500,33 @@ pub const Thread = struct {
         };
         return @intCast(usize, count);
     }
+
+    pub fn getCurrentThreadId() u64 {
```

Comment:
How is `getCurrentThreadId` different to `getCurrentId`?

Reply:
Good question. `getCurrentId` returns a `pthread_t` whenever its linked against libc and that's not the numeric thread ID but only a pointer. This function tries hard to squeeze a good looking ID that can be used for logging purposes.

The naming is a bit unfortunate, any suggestion?

Reply:
> Good question. `getCurrentId` returns a `pthread_t` whenever its linked against libc and that's not the numeric thread ID but only a pointer. This function tries hard to squeeze a good looking ID that can be used for logging purposes.

This would be at least useful to capture in a doc-comment.
As is I don't know how a user coming along is meant to know when to use which.

Reply:
Does `getCurrentIntegralId` sound better?